app/models/pulitzer/content_element.rb

```diff
@@ -44,11 +47,63 @@ def html
     end
 
     def empty_body?
-      image_type? ? !image? : body.blank?
+      !has_content?
     end
 
     def has_content?
-      image_type? ? image? : body.present?
+      if image_type?
+        image?
+      elsif custom_type?
+        custom_content.present?
+      else
+        body.present?
+      end
+    end
+
+    def custom_content
+      custom_option.value
+    end
+
+    def custom_display
+      "#{custom_option.display} #{custom_option.custom_option_list.name.singularize}"
+    end
+
+    def content
+      if custom_type?
+        custom_content
+      else
+        body
+      end
+    end
+
+    def content_display
+      if custom_type?
+        custom_display
+      else
+        body
+      end
+    end
+
+    def custom_type?
+      custom_option.present?
+    end
+
+    def clickable_kind
+      if custom_type?
+        custom_option.custom_option_list
+      elsif body.present?
+        Pulitzer::PostTypeContentElementType::Url_clickable
```

Comment:
Here you are calling Url_clickable but below, it's defined as URL_clickable (notice the capitalization)